cpm

```diff
@@ -248,13 +248,15 @@ _dnf() {
     remove)  $SUDO dnf remove "$@";;
     list)    dnf list --installed;;
     count)   rpm -qa | tot;;
-    update)  $SUDO dnf check-update;;
-    upgrade) $SUDO dnf check-update && $SUDO dnf distro-sync;;
+    update)  echo "No DNF equivalent or smth tell me what to place here";;
```

Comment:
Probably just mention that DNF doesn't support it; copy a message from another package manager.